apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -70,6 +87,16 @@ export default class AmazonFutureEngineerEligibilityForm extends React.Component
     this.setState(change);
   };
 
+  handleRoleChange = change => {
+    this.setState({professionalRole: change.target.value});
+  };
+
+  handleMultiSelectGradeBands = index => {
+    let gradeBands = this.state.gradeBands;
+    gradeBands[index] = !gradeBands[index];
```

Comment:
Sorry, I should've caught this in the previous PR but updating arrays in state require a copy (more info in [React docs](https://beta.reactjs.org/learn/updating-arrays-in-state#updating-arrays-without-mutation)). I think you could do something like:
```
const gradeBands = {...this.state.gradeBands};
gradeBands[index] = !gradeBands[index];
this.setState(gradeBands);
```